### DISCUSSION TITLE
Is it possible to hide icon from top bar??

### DISCUSSION THREAD

**Discussion by Jamir-boop** [FelipeFTN/Emoji-Copy#55] [Q&A]
Great project! As the title mentions, I'm interested in whether you could develop a feature or give me a little guide on how to add it myself. I haven’t developed an extension before.

**Comment by pavinjosdev** [comment]
Hi! 👋

The extension icon in the top-bar can be removed from its settings, you should download an "Extension Manager" app if you don't have one already. There is one by the same name available as a Flatpak:
https://flathub.org/apps/com.mattjakeman.ExtensionManager
